src/python/dxpy/bindings/dxapplet.py

```diff
@@ -58,11 +58,13 @@ def _get_run_input_common_fields(executable_input, **kwargs):
             if kwargs.get(arg) is not None:
                 run_input[arg] = kwargs[arg]
 
-        if kwargs.get('instance_type') is not None or kwargs.get('cluster_spec') is not None or kwargs.get('fpga_driver') is not None:
+        if any(kwargs.get(key) is not None for key in
+               ['instance_type', 'cluster_spec', 'fpga_driver', 'nvidia_driver']):
```

Comment:
please put it to a single line

Reply:
Yeah, I think the editor split it as the line is longer than 120chars, but I checked and its 121 so I guess its still OK to have it on 1 line.